api/swagger/swagger.yml

```diff
@@ -1,16 +1,134 @@
 consumes:
-- application/json
+  - application/json
 info:
   description: Thyra HTTP server API.
   title: thyra-server
   version: 0.0.0
 produces:
-- application/json
+  - application/json
 schemes:
-- http
-- https
-swagger: "2.0"
+  - http
+  - https
+swagger: '2.0'
 paths:
+  /mgmt/wallet:
+    get:
+      operationId: mgmtWalletGet
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Wallets retrieved
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Wallet'
+        '400':
+          description: Bad request.
+          schema:
+            $ref: '#/definitions/Error'
+        '422':
+          description: Unprocessable Entity - wallet.json file is corrupted.
```

Comment:
This looks very specific. Which means that when we will use this HTTP code for something else, we will need to update the swagger file...